tests/e2e/network-activated.spec.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { test, expect } from '@wordpress/e2e-test-utils-playwright';
+
+let page;
+
+test.describe.configure( { mode: 'serial' } );
+
+test.beforeAll( async ( { browser } ) => {
+	page = await browser.newPage();
+} );
+
+test.afterAll( async () => {
+	await page.goto( 'http://stream.wpenv.net/wp-admin/network/plugins.php' );
+	const isActive = await page
+		.getByLabel( 'Network Deactivate Stream' )
+		.isVisible();
+
+	if ( isActive ) {
+		// eslint-disable-next-line no-console
+		console.log( 'Deactivating Stream after Network tests.' );
+		await page.getByLabel( 'Network Deactivate Stream' ).click();
+	}
+} );
+
+test.describe( 'Network: shows network activated states', () => {
+	// Do we have a published row?
+	test( 'does not show stream in network admin when deactivated', async () => {
+		await page.goto( 'http://stream.wpenv.net/wp-admin/network/index.php' );
+		// Expects Stream log to not have the Network Settings.
+		await expect(
+			page.locator( '[href*="admin.php?page=wp_stream"]' ),
+		).not.toBeVisible();
+	} );
+
+	// We should not have an updated row. This times out which makes it fail.
+	test( 'does not have updated row', async () => {
+		await page.goto( 'http://stream.wpenv.net/wp-admin/network/plugins.php' );
+		const isActive = await page
```

Comment:
<!-- simple tasks that must be done -->
**🧹 chore:** I believe this should be the opposite:

```suggestion
		const isInactive = await page
```